### ISSUE TITLE
Fix raster hanging issue

### ISSUE THREAD

**Issue by JunAishima** [NSLS-II/lsdc#17]
Users and beamline scientists have reported that rasters occasionally hang.

**Comment by JunAishima** [comment]
first problem we found is that after vectorGo is triggered, the raster hangs. closer monitoring of PVs shows that VectorActive is set to 0 by the time the function vectorActiveWait starts polling for VectorActive to be 1. The vector program completes (such as during short line scans, as observed by beamline scientists) before the status check begins.

Stu Myers suggested removing the check as VectorActive should not be 1 before the vector program runs. The vector program also does not seem to take 0.15 sec to start.

Currently testing (will add commit reference)

**Comment by JunAishima** [comment]
second problem we have found is that even though LSDC goes through code to start vector (vectorGo, 1), the raster hangs.

I tested whether the PV object was still valid in LSDC by having the code 'setPvDesc("vectorGo", 1)' be run on the server. This triggered the collection.

A likely cause, suggested by Stu Myers, is that the zebra program is not finished by the time vectorGo is triggered. (The time taken between zebraVecDacSetup, when most zebra PVs are set, and vectorGo seems to be ~0.2 sec, which may not be enough for the zebra to finish.)

- wait 0.5 sec before vectorGo - see if problems happen less frequently as this will strongly suggest Zebra issues. If so, we will try to implement a more permanent solution later.
- set vectorGo to 0 early on so that we can camonitor the vectorGo PV to get timestamps

Notes: on AMX, the PV suffix for vectorGo: Cmd:Go-Cmd, for VectorActive: Sts:Running-Sts
